scripts/install

```diff
@@ -14,7 +14,7 @@ else
 fi
 
 function fetch_from_github(){
-  curl -s https://api.github.com/repos/verizon/nelson-cli/releases | grep browser_download_url | head -n 2 | cut -d '"' -f 4 | grep $PLATFORM
+  curl -s https://api.github.com/repos/verizon/nelson-cli/releases | grep browser_download_url | head -n 4 | cut -d '"' -f 4 | grep $PLATFORM
```

Comment:
I think this works, but it would be more robust if we grepped for platform first and then do `head -n 2`.  Then we'd be protected just in case we start publishing binaries for more platforms.